google/datalab/contrib/pipeline/airflow/airflow_vm_startup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+LOCAL_DAG_PATH=airflow/dags
+TEMP_CRON_FILE=temp_crontab.txt
+
+# TODO(rajivpb): These parameters should be gettable from the environment, i.e. they should have
+# been set by the VM's create script. So eventually, remove these lines.
+GCS_DAG_BUCKET="rajivpb-airflow-testing"
+GCS_DAG_PATH="/dags"
+
+export AIRFLOW_HOME=airflow
+
+apt-get --assume-yes install python-pip
+
+pip install airflow
+.local/bin/airflow initdb
```

Comment:
Unclear why you need the .local/bin. Doesn't pip install put it in the path? Or should you change $PATH to make sure airflow is on it?

Reply:
Curiously enough, pip install did not put it in the path. So I saw a need to do this. Didn't want to add it to the path and make changes to the environment variables since I was using this in only one place, so did it this way.

Reply:
See https://airflow.readthedocs.io/en/1.6.2-docs/start.html ... about AIRFLOW_HOME, and apparent lack of needing to use .local/bin or updating $PATH.